target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -389,6 +394,10 @@ ath79_setup_macs()
 		lan_mac=$(mtd_get_mac_ascii u-boot-env ethaddr)
 		label_mac=$(mtd_get_mac_ascii u-boot-env athaddr)
 		;;
+	engenius,ens202ext)
+		lan_mac=$(mtd_get_mac_binary art 0)
+		wan_mac=$(mtd_get_mac_binary art 0)
+		;;
```

Comment:
This is done in dts and can be dropped here entirely.

Reply:
So this part of the script is for handling the MAC if DTS cannot?

Reply:
exactly, e.g. when you have ascii format, and would use mtd_get_mac_ascii ...